improver/psychrometric_calculations/cloud_condensation_level.py

```diff
@@ -65,28 +64,29 @@ def __init__(self, model_id_attr: str = None):
         self.model_id_attr = model_id_attr
         self.temperature, self.pressure, self.humidity = None, None, None
 
-    def _make_ccl_cube(self, data: np.ndarray, pressure: np.ndarray) -> Cube:
+    def _make_ccl_cube(self, data: np.ndarray, is_temperature: bool) -> Cube:
         """Puts the data array into a CF-compliant cube"""
         attributes = {}
         if self.model_id_attr:
             attributes[self.model_id_attr] = self.temperature.attributes[
                 self.model_id_attr
             ]
+        if is_temperature:
+            name = "air_temperature_at_condensation_level"
+            units = "K"
+        else:
+            name = "air_pressure_at_condensation_level"
+            units = "Pa"
         cube = create_new_diagnostic_cube(
-            "air_temperature_at_condensation_level",
-            "K",
+            name,
+            units,
             self.temperature,
```

Comment:
I've added code to remove the height coord, and made sure that the inputs to the cloud top temperature are also updated for this.

Still thinking about the title.

Reply:
The `PostProcessingPlugin` class expects the output to be a `Cube`, rather than `Tuple[Cube, Cube]`, so it needs a small modification, which I will include here.

I've changed both the CloudCondensationLevel and CloudTopTemperature plugins to inherit PostProcessingPlugin. This also meant removing the temperature-on-pressure-levels cube from the mandatory attributes function call in CloudTopTemperature as "Post-Processed " is not included in the title attribute of that cube which resulted in it defaulting to "unknown" rather than raising an Error.